src/Tribe/Views/V2/Hooks.php

```diff
@@ -432,6 +437,13 @@ public function filter_redirect_canonical( $redirect_url = null, $original_url =
 			return $redirect_url;
 		}
 
+		// If the request is not for one of the enabled views, then bail.
+		$enabled_views = tribe_get_option( 'tribeEnableViews', array() );
```

Comment:
Lets use the manager method to get this: `Manager::get_publicly_visible_views`

Reply:
Ah, yes, good catch.